### PR TITLE
Hawkular 1259

### PR DESCRIPTION
**Pull request by jshaughn**
@jotak , Please review.  This is basically the 1259 branch rebased on master.  If you are OK with it I think we can merge (no squash to maintain the commit history).  After the merge we need to update the version to 5.0 in a follow-on PR.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 hawkular:
-  image: "hawkular/hawkular-services:latest"
+  # !! WARNING !! FIXME !! TODO !!
```
Comment (jotak) [inline docker-compose.yml]: So, what's the rightest docker tag now? Is it "hawkular-services:latest", for master?

**Comment by jotak** [comment]
Apart for the docker image to use (in theory it should be "latest", but I have a doubt given its size), it's LGTM in terms of code review. But I let @cfcosta or @israel-hdez tell if it's ok to merge in master.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+  # Reset to hawkular/hawkular-services:latest before merging in master
+  image: "hawkular/hawkular-services:hawkular-1259"
```
Comment (josejulio) [inline docker-compose.yml]: I think this can be updated to hawkular-services:latest before we forget about this

Comment (jshaughn) [reply docker-compose.yml]: pushed fix.

**Comment by josejulio** [comment]
LGTM

**Comment by jotak** [comment]
Thanks @jshaughn

**Pull request merged**